data/strings/sound.txt

```diff
@@ -1507,7 +1507,7 @@
     nb = Ta åttende avkjøring.
     nl = Neem de achtste afslag.
     pl = Zjedź ósmym zjazdem.
-    pt = Saia na oitava saída.
+    pt = Pegue a oitava saída.
     pt-BR = Saia na oitava saída.
```

Comment:
pt and pt-BR should be swapped here, right?

Reply:
Yep, my mistake, thanks for the fix and the merge!